lib/util/cache/package/redis.spec.ts

```diff
@@ -26,4 +30,106 @@ describe('util/cache/package/redis', () => {
       );
     });
   });
+
+  describe('init', () => {
+    beforeEach(() => {
+      vi.clearAllMocks();
+      vi.mocked(createClient).mockReturnValue({
+        connect: vi.fn(),
+      } as unknown as RedisClientType);
```

Comment:
use our partial helper method

https://github.com/renovatebot/renovate/blob/643d4893acef077dd719c30efd062089a38ee7fc/test/util.ts#L25-L30

Reply:
Do you have a specific suggestion? I tried a quick change and it failed